packages/api/src/entity/subscription.ts

```diff
@@ -7,12 +7,23 @@ import {
   PrimaryGeneratedColumn,
   UpdateDateColumn,
 } from 'typeorm'
-import { SubscriptionStatus, SubscriptionType } from '../generated/graphql'
 import { NewsletterEmail } from './newsletter_email'
 import { User } from './user'
 
 export const DEFAULT_SUBSCRIPTION_FOLDER = 'following'
 
+export enum SubscriptionStatus {
+  Active = 'ACTIVE',
+  Deleted = 'DELETED',
+  Unsubscribed = 'UNSUBSCRIBED',
+  RefreshError = 'REFRESH_ERROR',
```

Comment:
oh shoot, i have a feeling this could cause issues in other places where we filter based on status. We might need to do ACTIVE || REFRESH_ERROR. I wonder if it should be a separate property.

Actually, maybe it should be a timestamp? refresh_error_at and then we know when it went into error state.

Reply:
good catch! so we filter subscriptions based on status = 'active' in some clients? then a separate column makes sense.

yeah, i agree with `refresh_error_at` and we could do a `error_count` as well